packages/node_modules/@webex/internal-plugin-ediscovery/test/unit/spec/content.js

```diff
@@ -195,6 +195,37 @@ describe('EDiscovery Content API Tests', () => {
 
       return result;
     });
+
+    it('GetContentSummary submits an audit log request', () => {
+      const result = webex.internal.ediscovery.postAuditLog(uuid, {timeoutMs: 1000})
+        .then((res) => {
+          expect(res.statusCode).equal(200);
+          sinon.assert.calledWith(webex.request, sinon.match.has('timeout', 1000));
+        });
+
+      return result;
+    });
+
+    it('GetContentSummary submits a malformed audit request and gets rejected', () => {
+      const noUuid = '';
+      const result = expect(webex.internal.ediscovery.postAuditLog(noUuid, {timeoutMs: 1000})).to.be.rejectedWith(Error, 'Undefined parameter');
+
+      return result;
```

Comment:
Why is this being returned?

Reply:
I was following the pattern of other tests, i presumed returning it was required for the test framework somehow. I can remove this.